### ISSUE TITLE
TODO: Add title

### ISSUE THREAD

**Issue by donletag** [koalaman/shellcheck#2066]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2002, SC2162, SC2034
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash
path=Documents/script/
cd $path || exit

unzip_and_replace(){
    file=$1
    unzip "$file"
    cat 'Annonces.csv' | while read line; do
        sed -i -e "s/!#/,/g" 'Annonces.csv'
    done

    cp -f 'Annonces.csv' "$file"'_new.csv'
    rm -f 'Annonces.csv'
}

unzip_and_replace "portissim.zip"
```

#### Here's what shellcheck currently says:
```
Line 8	SC2002: Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead.
Line 8	SC2162: read without -r will mangle backslashes.
Line 8	SC2034: line appears unused. Verify use (or export if used externally).
```
#### Here's what I wanted or expected to see:

TODO: Describe expected/desired output
